tools/build_variables.bzl

```diff
@@ -462,6 +463,7 @@ libtorch_python_core_sources = [
     "torch/csrc/jit/passes/onnx/scalar_type_analysis.cpp",
     "torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp",
     "torch/csrc/jit/passes/onnx/prepare_inplace_ops_for_onnx.cpp",
+    "torch/csrc/jit/passes/onnx/shape_inference.cpp",
```

Comment:
We are doing both shape and type inference in this component. We should consider renaming the filename here and other sites to include "type" in the name. So something like `shape_type_inference.cpp`.